### ISSUE TITLE
Does not start on Tomcat 8.5.8 (+ some test cases are fail on Tomcat 8.5.9)

### ISSUE THREAD

**Issue by kazuki43zoo** [spring-projects/spring-boot#7615]
I tried the Tomcat 8.5.8.

```xml
<tomcat.version>8.5.8</tomcat.version>
```

However it does not start the tomcat server.

```
...
2016-12-11 03:12:06.364 ERROR 20786 --- [           main] o.a.coyote.http11.Http11NioProtocol      : Failed to start end point associated with ProtocolHandler [http-nio-8080]

java.net.BindException: Address already in use
	at sun.nio.ch.Net.bind0(Native Method) ~[na:1.8.0_102]
	at sun.nio.ch.Net.bind(Net.java:433) ~[na:1.8.0_102]
...
```

I tried to change port number as follow:

```properties
server.port=18080
```

A server is started, However ... tomcat is started with two ports(8080 and 18080) as follow:

```
...
2016-12-11 03:15:39.717  INFO 20842 --- [           main] s.b.c.e.t.TomcatEmbeddedServletContainer : Tomcat started on port(s): 8080 (http) 18080 (http)
2016-12-11 03:15:39.721  INFO 20842 --- [           main] com.example.Tomcat858DemoApplication     : Started Tomcat858DemoApplication in 2.639 seconds (JVM running for 3.076)
```

This behavior is resolved at Tomcat 8.5.9, however some test cases are fail.
For details see https://travis-ci.org/spring-projects/spring-boot/builds/182846310.

## Versions

1.4.2.RELEASE

**Comment by bclozel** [comment]
Hi @kazuki43zoo 
I think that the first part of your report (Tomcat 8.5.8 not starting) is related to https://github.com/spring-projects/spring-boot/issues/7360#issuecomment-260286324 and https://bz.apache.org/bugzilla/show_bug.cgi?id=60368 - which are fixed in 8.5.9.

**Comment by kazuki43zoo** [comment]
Hi @bclozel , thanks for introduce bug information !!
Oh... it is reported by @wilkinsona :) 👍

**Comment by philwebb** [comment]
We'll fix the test failures when #7360 is closed.
